static/app/utils/growthAnalyticsEvents.tsx

```diff
@@ -8,6 +10,30 @@ type ShowParams = MobilePromptBannerParams & {
   mobileEventClientOsName: string;
 };
 
+type PlatformParam = {
+  platform: PlatformKey;
+};
+
+type PlatformCategory = {
+  category: string;
+  source?: string;
+};
+
+type PlatformPickerParam = {
+  platformId: string;
```

Comment:
@roggenkemper forgot to mention this earlier, but we should use `snake_case` for event parameters (so `platform_id` here).